src/main/java/com/google/api/codegen/gapic/MainGapicProviderFactory.java

```diff
@@ -72,8 +72,8 @@
   public static final String NODEJS = "nodejs";
   public static final String NODEJS_DOC = "nodejs_doc";
   public static final String PHP = "php";
-  public static final String PYTHON = "python";
-  public static final String PYTHON_DOC = "python_doc";
+  public static final String PYTHON = "py";
+  public static final String PYTHON_DOC = "py_doc";
```

Comment:
Does PYTHON_DOC even need to exist anymore?

Reply:
We should probably kill this, since it is replaced by your plugin.

Reply:
Done